examples/semver.peggy

```diff
@@ -0,0 +1,60 @@
+/**
+ * SemVer.org v2
+ * https://semver.org/spec/v2.0.0.html
+ * For unit tests see: https://github.com/dselman/peggy-semver
+ */
+semver
+  = versionCore:versionCore
+    pre:('-' @preRelease)?
+    build:('+' @build)?
+  {
+    return { versionCore, pre, build };
```

Comment:
Don't think that hiding most important information into a deeper object with non-descript name is a good idea. At least `{...versionCore, pre, build}`, please.